docker/Dockerfile.test

```diff
@@ -8,14 +8,19 @@ RUN rm -rf /var/lib/apt/lists/*
 ENV WORKPATH /go/src/github.com/openark/orchestrator
 WORKDIR $WORKPATH
 
-RUN curl -O "https://raw.githubusercontent.com/openark/orchestrator-ci-env/master/bin/linux/dbdeployer.gz"
-RUN curl -O "https://raw.githubusercontent.com/openark/orchestrator-ci-env/master/mysql-tarballs/mysql-5.7.26.tar.xz"
-RUN gunzip ./dbdeployer.gz
-RUN chmod +x ./dbdeployer
-RUN mkdir -p ./sandbox/binary
+# Setup dbdeployer
+RUN mkdir /dbdeployer
+RUN (cd /dbdeployer && wget https://github.com/datacharmer/dbdeployer/releases/download/v1.64.0/dbdeployer-1.64.0.linux.tar.gz)
+RUN (cd /dbdeployer && tar -xf dbdeployer-1.64.0.linux.tar.gz)
+RUN (cd /dbdeployer && ln -s dbdeployer-1.64.0.linux dbdeployer)
+RUN (cd /dbdeployer && ./dbdeployer defaults update reserved-ports '0')
 
-RUN ./dbdeployer unpack mysql-5.7.26.tar.xz --sandbox-binary $WORKPATH/sandbox/binary
+# For dev purposes only, just to avoid downloading over and over via download-mysql script
+# RUN (mkdir /mysql-tarballs-downloaded)
+# COPY docker/Percona-Server-8.0.26-16-Linux.x86_64.glibc2.12-minimal.tar.gz /mysql-tarballs-downloaded/ 
 
 COPY . .
+RUN (cd /dbdeployer && ./dbdeployer defaults update reserved-ports '0')
 
-CMD ["script/test-all"]
+ENV TARBAL_URL_DEFAULT https://raw.githubusercontent.com/kamil-holubicki/orchestrator-ci-env/master/mysql-tarballs/mysql-5.7.26.tar.xz
```

Comment:
May be it's worth having a default archive not in private repo, but in some of our repos ?

Reply:
The reason why it is done this way is that the upstream has this binary present in orchestrator-ci-env repo and then its path is hard coded in scripts. I changed the functionality to be able to provide any tarball but wanted to keep the default behavior in place, so fallback to this particular tarball. I agree with your proposal, but at this stage, I wanted to integrate everything with as small changes as possible.
We can change it in the future if needed.
BTW I changed it from my fork to official Percona repo as we have it forked now